recipes/kalpy/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+c_compiler:                    # [win]
+  - vs2019                     # [win]
+cxx_compiler:                  # [win]
+  - vs2019                     # [win]
```

Comment:
Delete this file (we've moved to default vs2019 already across all of conda-forge)